docs/framework/release-notes/2024/08-13-august-security-and-quality-rollup.md

```diff
@@ -0,0 +1,99 @@
+---
+title: August 2024 security and quality rollup
+description: Learn about the improvements in the .NET Framework August 2024 security and quality rollup.
+ms.date: 08/13/2024
+---
+# August 2024 security and quality rollup
+
+_Released August 13, 2024_
+
+## Summary of what is new in this release
+
+- [Security improvements](#security-improvements)
+- [Quality and reliability improvements](#quality-and-reliability-improvements)
+
+### Security improvements
+
+#### CVE-2024-29187 - Elevation of Privilege vulnerability
+
+This security update addresses an elevation of privilege vulnerability detailed in [CVE 2024-29187](https://msrc.microsoft.com/update-guide/vulnerability/CVE-2024-29187).
+
+#### Defense in depth vulnerability
+
+This security update addresses an issue with serialization binders in BinaryFormatter.
+
+### Quality and reliability improvements
+
+This release contains the following quality and reliability improvements.
+
+#### .NET fundamentals
+
+Addresses an issue to render WU automation ListView sub items. (*Applies to: .NET Framework 4.8, 4.8.1.*)
```

Comment:
What is WU?

Reply:
looks like this mistakenly got pasted into this blurb.  I'll remove "WU automation" from this link

Reply:
Also, WU is Windows shorthand for "Windows Update".